project/src/components/player-screen/player-screen.tsx

```diff
@@ -1,17 +1,47 @@
-function PlayerScreen(): JSX.Element {
+import { useHistory, useParams } from 'react-router-dom';
+import type { ParamsWithId, Film } from '../../types/types';
+import { formatElapsedTime } from '../../utils/date';
+
+type PlayerScreenProps = {
+  films: Film[],
+}
+
+
+function PlayerScreen({films}: PlayerScreenProps): JSX.Element {
+  const getFilmById = (id: number) => {
+    const foundFilm = films.find((film) => film.id === id);
+
+    if (!foundFilm) {
+      throw new Error(`Film with id=${id} does not exist`);
+    }
+
+    return foundFilm;
+  };
```

Comment:
```suggestion
  const getFilmById = (offers, id: number) => {
    const foundFilm = films.find((film) => film.id === id);

    if (!foundFilm) {
      throw new Error(`Film with id=${id} does not exist`);
    }

    return foundFilm;
  };
```
такие штуки лучше выносить из компонентов, чтобы они были чище